doc/en/announce/release-7.0.0rc1.rst

```diff
@@ -0,0 +1,88 @@
+pytest-7.0.0rc1
+=======================================
+
+pytest 7.0.0rc1 has just been released to PyPI.
+
+This is a bug-fix release, being a drop-in replacement. To upgrade::
```

Comment:
Ugh, lies... This is because the release script does:

https://github.com/pytest-dev/pytest/blob/1d6755340348a8ca538313309609e8fb7036b224/scripts/release.py#L25-L27

Reply:
time to parse it with packaging  to manage th e parts better

Reply:
I'd rather just pass the `--major` flag we pass to `prepare-release-pr.py` already through to `release.py` instead of relying on parsing the version number to guess information we already have.